hash/msx1_cart.xml

```diff
@@ -27588,8 +27433,7 @@ legacy FM implementations cannot find it.
 	<software name="z80rogue">
 		<description>z80rogue</description>
 		<year>2019</year>
-		<publisher>&lt;homebrew&gt;</publisher>
-		<info name="developer" value="Oscar Toledo Gutierrez"/>
+		<publisher>Oscar Toledo Gutierrez</publisher>
```

Comment:
If this is the chess programmer, it’s “Óscar Toledo Gutiérrez”: https://www.chessprogramming.org/%C3%93scar_Toledo_Guti%C3%A9rrez

Reply:
Yes, this is the same person. The software was published under the same github account.

Reply:
You missed more instances of his name, e.g. on Zombie Near.

Also, should César Rincón be credited as developer on “MSX Pong!”?

Reply:
I fixed that one as well. Btw, he did spell his own name as 'cesar rincon' on that title.
<img width="263" alt="Screenshot 2023-08-02 at 18 18 36" src="https://github.com/mamedev/mame/assets/9140441/2422ccb8-5382-4631-8a17-66d9e11c3c29">